tensorflow/compiler/xla/service/gpu/ir_emitter.cc

```diff
@@ -205,8 +205,9 @@ bool IrEmitter::MaybeEmitDirectAtomicOperation(
   }
 
   if (root_opcode == HloOpcode::kAdd) {
+    llvm::Triple target_triple = llvm::Triple(module_->getTargetTriple());
     // NVPTX supports atomicAdd on F32 and integer types.
-    if (element_type == F32) {
+    if (target_triple.isNVPTX() && (element_type == F32)) {
```

Comment:
Nit: No need for extra parenthesis around `element_type == F32`.

Reply:
removed.